src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/broadcast/broadcast_kernel_base.cpp

```diff
@@ -31,6 +31,29 @@ BroadcastKernelBase::DispatchData BroadcastKernelBase::SetDefault(const broadcas
     return dispatchData;
 }
 
+static std::string GetInputBlockND(const broadcast_params& params, int num) {
+    const auto& input = params.inputs[num];
```

Comment:
This num is always zero?

Reply:
Originally I put the num argument so that I could get the blockND for any input. But in the case of broadcast right now, yes, since I only needed the blockND for input[0]. I left the code as it was just to make it more general then to make the function return only the blockND for input[0]. Do you recommend that I remove the num argument for broadcast?